File: tests/fixtures/py/module_scope/src/shadowed.py
from stripe import StripeClient

client = StripeClient("sk_test")

charge = client.charges.create(amount=100, currency="usd")
print(charge.status)


def summarise():
    charge = compute_locally()
    return charge.total


def compute_locally():
    return {"total": 42}

File: tests/fixtures/ts/module_scope/src/shadowed.ts
import Stripe from 'stripe';

const stripe = new Stripe(process.env.STRIPE_KEY!);

const charge = await stripe.charges.create({ amount: 100, currency: 'usd' });
console.log(charge.status);

export function summarise() {
  const charge = computeLocally();
  return charge.total;
}

function computeLocally() {
  return { total: 42 };
}
